### DISCUSSION TITLE
Cursor does not allow you to select items in a P-Dropdown in v17.3.3

### DISCUSSION THREAD

**Discussion by hepple** [primefaces/community#969] [PrimeNG] [answered]
I have the below, as well as many other examples, where in v16 the cursor would allow you to select an item. This has stopped working in v17+

`            <div class="col-4">
                <span class="p-float-label">
                    <p-dropdown formControlName="newGovernorSalutationId" [options]="siSalutations" id="newGovernorSalutationId" [filter]="true" filterBy="label" optionLabel="label" optionValue="value" appendTo="body"></p-dropdown>
                    <label for="newGovernorSalutationId">Salutation</label>
                </span>
            </div>`

**Comment by cetincakiroglu** [comment]
Hi,

Please provide a stackblitz example so we can identify the issue clearly?

**Reply by hepple** [reply, marked as answer]
I've figured out it was actually accepting the keystrokes but the item wasn't highlighting.

I've added this to css and it's fixed it. There was no conflicting css that I could see.

.p-dropdown-panel .p-dropdown-items .p-dropdown-item.p-focus {
    background: #e0e0e0;
}
